dashel/dashel-common.cpp

```diff
@@ -59,83 +59,67 @@
 namespace Dashel
 {
 	using namespace std;
-	
+
 	// frome dashe-private.h
-	ExpandableBuffer::ExpandableBuffer(size_t size) :
-		_data((unsigned char*)malloc(size)),
-		_size(size),
-		_pos(0)
-	{
-	}
-	
+	ExpandableBuffer::ExpandableBuffer(size_t size) : _data((unsigned char*)malloc(size)), _size(size), _pos(0) {}
```

Comment:
In this case, putting all initialized members on a single line reduces readability.

Reply:
That can be tweaked